.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
-*       @pokey @AndreasArvidsson @phillco
+* @cursorless-dev/code-owners
 
-*keyboard* @pokey @AndreasArvidsson @phillco @josharian
-*Keyboard* @pokey @AndreasArvidsson @phillco @josharian
+*keyboard* @cursorless-dev/code-owners @josharian
+*Keyboard* @cursorless-dev/code-owners @josharian
 
-*neovim* @pokey @AndreasArvidsson @saidelike @fidgetingbits
-*Neovim* @pokey @AndreasArvidsson @saidelike @fidgetingbits
+*neovim* @cursorless-dev/code-owners @saidelike @fidgetingbits
+*Neovim* @cursorless-dev/code-owners @saidelike @fidgetingbits
```

Comment:
Nice, thanks @pokey. I was debating mentioning that I wasn't a code owner for neovim in the previous version, which I wasn't sure was intentional or not (either way is fine, although I'm actually kind of interested in reviewing their code, lua is neat). The team definition prevents mistakes

cc @AndreasArvidsson